src/mbed_tools/cli/build.py

```diff
@@ -23,7 +23,7 @@
     help="The toolchain you are using to build your app.",
 )
 @click.option("-m", "--mbed-target", help="A build target for an Mbed-enabled device, e.g. K64F.")
-@click.option("-b", "--build-type", default="develop", help="The build type (release, develop or debug).")
+@click.option("--profile", default="develop", help="The build type (release, develop or debug).")
```

Comment:
Should we keep "-b" for this?

Reply:
Usually, first letter is used as an alias. But we are already using `-p` for `--program-path`. So I thought of removing `-b`.

Reply:
Since backward compatibility can be achieved through `-b` option, should I change the `major` to `bugfix`?

Reply:
The full option changed, which breaks backwards compatibility still.